codegen/smithy-aws-go-codegen/src/main/java/software/amazon/smithy/aws/go/codegen/AwsClientUserAgent.java

```diff
@@ -41,13 +41,18 @@ public void writeAdditionalFiles(
             GoDelegator goDelegator
     ) {
         ServiceTrait serviceTrait = settings.getService(model).expectTrait(ServiceTrait.class);
+        String servceId = serviceTrait.getSdkId().replace("-", "").replace(" ", "").toLowerCase();
```

Comment:
Nit: Java replace takes a regex. Right now it will traverse that string thrice I think.